.github/workflows/InitDev Postgres, MultiTenancy.yml

```
@@ -33,42 +33,53 @@ env:
   LANG: "en_US.utf8"
 
 jobs:
-  FindStandardVersions:
-    runs-on: ubuntu-latest    
-    outputs:
-      StandardVersions: ${{ steps.Set_StandardVersions.outputs.StandardVersions }}
-    steps:
-    - name: Checkout Ed-Fi-ODS-Implementation
-      uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
-      with:
-          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation
-          path: Ed-Fi-ODS-Implementation/    
-    - name: Checkout Ed-Fi-ODS
-      uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
-      with:
-          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS
-          path: Ed-Fi-ODS/
-    - name: Is pull request branch exists in Ed-Fi-ODS
-      working-directory: ./Ed-Fi-ODS-Implementation/
-      shell: pwsh
-      run: |
-           .\build.githubactions.ps1 CheckoutBranch -RelativeRepoPath "../Ed-Fi-ODS"          
-    - name: Set StandardVersions
-      id: Set_StandardVersions
-      working-directory: ./Ed-Fi-ODS/
-      run: |
-        $output = .\build.githubactions.ps1 StandardVersions -ProjectFile "$env:GITHUB_WORKSPACE/Ed-Fi-ODS/Application/EdFi.Ods.Standard/EdFi.Ods.Standard.csproj" | Out-String
-        echo "StandardVersions=$output" >> $env:GITHUB_OUTPUT
-        Write-host "StandardVersions is  $output"
-      shell: pwsh
+  FindStandardAndExtensionVersions:
+     runs-on: ubuntu-latest
+     outputs:
+       StandardVersions: ${{ steps.Set_StandardVersions.outputs.StandardVersions }}
+       ExtensionVersions: ${{ steps.Set_ExtensionVersions.outputs.ExtensionVersions }}
+     steps:
+     - name: Checkout Ed-Fi-ODS
+       uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
+       with:
+           repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS
+           path: Ed-Fi-ODS/
+     - name: Checkout Ed-Fi-Extensions
+       uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4
+       with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-Extensions
+          path: Ed-Fi-Extensions/
+     - name: Is pull request branch exists in Ed-Fi-Extensions
+       working-directory: ./Ed-Fi-ODS/
+       shell: pwsh
+       run: |
+         .\build.githubactions.ps1 CheckoutBranch -RelativeRepoPath "../Ed-Fi-Extensions"
+     - name: Set StandardVersions
+       id: Set_StandardVersions
+       working-directory: ./Ed-Fi-ODS/
+       run: |
+         $output = .\build.githubactions.ps1 StandardVersions -ProjectFile "$env:GITHUB_WORKSPACE/Ed-Fi-ODS/Application/EdFi.Ods.Standard/EdFi.Ods.Standard.csproj"
+         echo "StandardVersions=$output" >> $env:GITHUB_OUTPUT
+         Write-host "StandardVersions is  $output"
```

In fact, this `echo` command does not work as written in PowerShell Core. I just tested locally and this does not set a value to `$env:GITHUB_OUTPUT`.

```suggestion
          $env:GITHUB_OUTPUT = "StandardVersions=$output"
         Write-host "StandardVersions is  $output"
```